packages/code-du-travail-data/dataset/themes/index.js

```diff
@@ -0,0 +1,92 @@
+const cstojson = require("csvtojson");
+const slugify = require("slugify");
+
+/**
+ *
+ * flattenThemes take a csv file of themes
+ * [{
+ *    niveau1: '1 - Recrutement et contrat de travail',
+ *    niveau2: '9 - Recrutement',
+ *    niveau3: '50 - Méthodes de recrutement',
+ *    niveau4: ''
+ * }, {
+ *    niveau1: '1 - Recrutement et contrat de travail',
+ *    niveau2: '9 - Recrutement',
+ *    niveau3: '51 - Curriculum vitae (CV)',
+ *    niveau4: ''
+ * }]
+ *
+ *
+ *  and will return a flat list of theme
+ * {
```

Comment:
Does the output here matches the sample above ?? I'm not sure I get the logic if so

Reply:
missing 50 and 51 in the output

Reply:
Ok got it !